ui/js/component/video/view.jsx

```diff
@@ -119,6 +119,8 @@ class Video extends React.Component {
     if (mediaType === "video") {
       klassName += "video-embedded video"
       klassName += isPlaying ? " video--active" : " video--hidden"
+    } else if (mediaType === "application") {
+      klassName += "video-embedded"
```

Comment:
Is all of the class checking correct here? I may not understand the conditions properly, but seems like you could end up with repeated classes and/or classes stacked without a space.